manifests/install.pp

```diff
@@ -3,7 +3,7 @@
   $ensure  = 'latest',
   ) {
 
-  package { 'scout':
+  package { '^scout':
```

Comment:
@denmat I can't tell by looking at that code if it fixes this issue and even if it did I think it would warrant a separate tests.